persistence/sql/persister_oauth2.go

```diff
@@ -252,10 +253,20 @@ func (p *Persister) deleteSessionBySignature(ctx context.Context, signature stri
 	signature = p.hashSignature(signature, table)
 
 	/* #nosec G201 table is static */
-	return sqlcon.HandleError(
-		p.Connection(ctx).
-			RawQuery(fmt.Sprintf("DELETE FROM %s WHERE signature=?", OAuth2RequestSQL{Table: table}.TableName()), signature).
-			Exec())
+	if err := p.Connection(ctx).
+		RawQuery(fmt.Sprintf("DELETE FROM %s WHERE signature=?", OAuth2RequestSQL{Table: table}.TableName()), signature).
+		Exec(); errors.Is(err, sql.ErrNoRows) {
```

Comment:
This can never be true because `Exec()` is no longer wrapped in `sqlcon.HandleError`